src/infection-model/seir.tsx

```diff
@@ -113,14 +127,34 @@ function simulateOneDay(inputs: SimulationInputs & SingleDayInputs) {
     exposedDelta = 0;
     susceptibleDelta = 0;
   } else {
-    exposedDelta =
-      Math.min(
-        susceptible,
-        (beta * totalInfectious * susceptible) / totalPopulation,
-      ) -
-      alpha * exposed;
-    susceptibleDelta =
-      (-beta * totalInfectious * susceptible) / totalPopulation;
+    if (simulateStaff) {
```

Comment:
I don't get at first glance why the more complex calculation is gated by us not "simulating staff." What obvious point am I missing here?

Reply:
my understanding is that the effect of facility type on staff is not as strong as it is on the incarcerated, so the staff modeling leaves some of that out ... @justkunz would know better than I would though

Reply:
The intuition here was that the rate of spread for the faculty does not change under different facility types. I think an argument could be made that faculty have completely different R0 values, but at this point I doubt there is data to show that conclusively.

Reply:
Got it, that makes a lot of sense. Thanks!